tests/integrationtest/r/executor/index_merge_reader.result

```diff
@@ -530,3 +530,29 @@ select /*+ use_index_merge(t1, primary, idx1, idx2) */ c1 from t1 where c1 < 102
 Error 8175 (HY000): Your query has been cancelled due to exceeding the allowed memory limit for a single SQL query. Please try narrowing your query scope or increase the tidb_mem_quota_query limit and try again.[conn=<num>]
 set global tidb_mem_oom_action = DEFAULT;
 set @@tidb_mem_quota_query = default;
+drop table if exists t1;
+CREATE TABLE `tcfc8cb35` (`col_77` mediumint(9) NOT NULL,`col_78` year(4) NOT NULL,`col_79` varbinary(62) NOT NULL,`col_80` text COLLATE utf8mb4_unicode_ci NOT NULL,`col_81` tinyint(4) NOT NULL DEFAULT '115',`col_82` smallint(6) DEFAULT '2675',`col_83` date DEFAULT '1981-09-17',`col_84` mediumint(8) unsigned NOT NULL,`col_85` varchar(384) CHARACTER SET gbk COLLATE gbk_bin DEFAULT NULL,UNIQUE KEY `idx_23` (`col_84`,`col_82`),PRIMARY KEY (`col_84`,`col_77`) /*T![clustered_index] CLUSTERED */,UNIQUE KEY `idx_25` (`col_84`,`col_85`(5),`col_81`)) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin PARTITION BY HASH (`col_84`) PARTITIONS 1;
```

Comment:
It's better to refine the string values.

Reply:
case simplified